atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -429,6 +451,11 @@ def get_cluster_info(self, cluster, platform):
         if self.osbs_client_config:
             kwargs['conf_file'] = os.path.join(self.osbs_client_config, 'osbs.conf')
 
+        if platform in self.build_image_digests:
+            kwargs['build_image'] = self.build_image_digests[platform]
+        else:
+            raise RuntimeError("We don't have build_image for platform %s", platform)
```

Comment:
This error message sounds odd. Maybe: `build_image for platform '%s' not available`

Reply:
ok